fix: support resuming completed deployments (#762)
salesforcecli/plugin-deploy-retrieve: messages/deploy.metadata.resume.md

# summary

> Resume watching a deploy operation and update source tracking upon completion as necessary.

Reviewer: ```suggestion
Resume watching a deploy operation and update source tracking when the deploy completes.
```

# description

> Use this command to resume watching a deploy operation if the original command times out or you specified the --async flag. Deploy operations include standard deploys, quick deploys, deploy validations, and deploy cancellations. This command doesn't resume the original operation itself, because the operation always continues after you've started it, regardless of whether you're watching it or not. Upon completion, source tracking information will be updated as necessary.

Reviewer: ```suggestion
Use this command to resume watching a deploy operation if the original command times out or you specified the --async flag. Deploy operations include standard deploys, quick deploys, deploy validations, and deploy cancellations. This command doesn't resume the original operation itself, because the operation always continues after you've started it, regardless of whether you're watching it or not. When the deploy completes, source tracking information is updated as needed.
```

Show concise output of the deploy operation result.

# warning.DeployNotResumable

> Job ID %s is not resumable since it already completed with status: %s. Displaying results...

Reviewer: ```suggestion
Job ID %s is not resumable because it already completed with status: %s. Displaying results...
```
